framework/src/modules/chain/blocks/blocks.js

```diff
@@ -452,23 +464,27 @@ class Blocks extends EventEmitter {
 
 	async _receiveBlockFromNetworkV2(block) {
 		// Current time since Lisk Epoch
-		// Better to do it here rather than in the Sequence so receiving time is more accurate
-		const newBlockReceivedAt = this.slots.getTime();
+		block.receivedAt = Math.floor(this.slots.getTime() / 1000);
```

Comment:
Please use `slots.getSlotTime()` which will make it consistent to the forging timestamp. So what ever value it returns in seconds or miliseconds it will be consistent across code.